jobs/build-webapp.groovy

```diff
@@ -528,6 +528,18 @@ def uploadGraphqlSafelist() {
 }
 
 
+def createCloudRunTags(){
+   echo("Creating Cloud Run tags.")
+   dir("webapp") {
+      try {
+         exec(["deploy/update_cloud_run_tags.py", NEW_VERSION]);
+      } catch (e) {
+         echo("Failed to wait for new version: ${e}");
```

Comment:
I think this message can be updated. :-)

But I think you should get rid of the try/catch entirely.  This seems like a fatal error to me (since it means we can't talk to our cloud-run services once we set-default.)

Reply:
ha! yes :D